website/general_issues.qmd

```diff
@@ -15,6 +15,10 @@ Where do they end up and who will look at them etc... -->
 
 <!-- I suggest writing a short overview of the most common and important fields of the DESCRIPTION -->
 
+## Description Length
+
+The description field in the DESCRIPTION file should be a short paragraph (2+ sentences) on the package's purpose, motivation and may include further references to the package documentation website. A good example of this description length is included in the ['tidyverse'](https://github.com/tidyverse/tidyverse/blob/main/DESCRIPTION#L8-L12) package.
+
```

Comment:
Possibly add in now or later a good analogy is to compare the Description length to a cover text (short blurb inside a book) or an abstract.